lib/elixir/test/elixir/calendar/holocene.exs

```diff
@@ -78,6 +78,59 @@ defmodule Calendar.Holocene do
     )
   end
 
+  @impl true
+  @spec inspect_date(Calendar.year(), Calendar.month(), Calendar.day(), Inspect.Opts.t()) ::
+          String.t()
+  def inspect_date(year, month, day, _) do
+    date_to_string(year, month, day)
+  end
+
+  @impl true
+  def inspect_datetime(
+        year,
+        month,
+        day,
+        hour,
+        minute,
+        second,
+        microsecond,
+        time_zone,
+        zone_abbr,
+        utc_offset,
+        std_offset,
+        _opts
+      ) do
+    datetime_to_string(
+      year,
+      month,
+      day,
+      hour,
+      minute,
+      second,
+      microsecond,
+      time_zone,
+      zone_abbr,
+      utc_offset,
+      std_offset
+    )
+  end
+
+  @impl true
+  def inspect_naive_datetime(year, month, day, hour, minute, second, microsecond, _opts) do
+    naive_datetime_to_string(year, month, day, hour, minute, second, microsecond)
+  end
+
+  @impl true
+  def inspect_time(
```

Comment:
This is not being used, a test for `Time` is missed

Reply:
Added a test for `inspect_time/2` and fixed the docs in line with comments above.